src/plugins/expressions/common/execution/execution.ts

```diff
@@ -382,7 +385,7 @@ export class Execution<
     const resolveArgFns = mapValues(argAstsWithDefaults, (asts, argName) => {
       return asts.map((item: ExpressionAstExpression) => {
         return async (subInput = input) => {
-          const output = await this.params.executor.interpret(item, subInput, {
+          const output = await this.params.executor.interpret(item, subInput, this.extraContext, {
```

Comment:
we extracted this fix into a separate PR:

https://github.com/elastic/kibana/pull/70795